lib/services/devices/deviceService.js

```diff
@@ -181,9 +181,6 @@ function mergeDeviceWithConfiguration(fields, defaults, deviceData, configuratio
     if (configuration && configuration.expressionLanguage && deviceData.expressionLanguage === undefined) {
         deviceData.expressionLanguage = configuration.expressionLanguage;
     }
```

Comment:
```suggestion
```

I understand this should be also removed.

(Maybe a grep search by `expressionLanguage` could help to identify other similar leftover we may have yet)

Reply:
Related https://github.com/telefonicaid/iotagent-node-lib/issues/1384

Reply:
IMHO Delete of expressionLanguague should be done in another PR, with all related stuff https://github.com/telefonicaid/iotagent-node-lib/issues/1384

Reply:
> IMHO Delete of expressionLanguague should be done in another PR, with all related stuff #1384

Done in https://github.com/telefonicaid/iotagent-node-lib/issues/1384